src/Test1.java

```diff
@@ -1,35 +1,37 @@
 import java.util.*;
+
 enum Operation
 {
     ADD, SUBTRACT, MULTIPLY, DIVIDE
 }
 public class Test1 {
 
-    public static double basicCalculator(Operation operation, int num1, int num2){
+    public static double basicCalculator(final Operation operation, final int num1, final int num2){
         double result = 0;
 
-        if(operation != Operation.ADD) {
-            result = num1 + num2;
-        }
-        else if(operation != Operation.SUBTRACT) {
-            result = num1 - num2;
-        }
-        else if(operation != Operation.MULTIPLY) {
-            result = num1 * num2;
+        switch(operation) {
+            case ADD:
+                // if operation = ADD, return the sum
+                result = num1 + num2;
+                break;
+            case SUBTRACT:
+                result = num1 - num2;
+                break;
+            case MULTIPLY:
+                result = num1 * num2;
+                break;
+            case DIVIDE:
+                if (num2 == 0) {
+                    System.out.println("Can't divide by 0");
+                } else {
+                    result = num1 / num2;
+                }
+                break;
         }
-        else {
-            if(num2 == 0) {
-                System.out.println("Error: divide by zero");
-            }
-            else {
-                result = num1 / num2;
```

Comment:
Can do a type conversion so it's not integer division?

Reply:
good catch will do!

Reply:
Cool!